assets/scss/projects.scss

```diff
@@ -1153,6 +1153,7 @@ html.modal-open, html.modal-open body {
 
       table {
         color: $govuk-text-colour;
+        margin-bottom: 0;
```

Comment:
Should this declaration be applied in the context of the overflow wrapper, so whenever the scrollbar is shown it always sits flush to the bottom of the table, or do we want to keep the gap when it's not against the purple background?